test/e2e/gardener/upgrade/upgrade.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("Gardener upgrade Tests for", func() {
 			f         = framework.NewShootCreationFramework(&framework.ShootCreationConfig{
 				GardenerConfig: e2e.DefaultGardenConfig(projectNamespace),
 			})
-			shootTest = e2e.DefaultShoot("e2e-upgrade-hib")
+			shootTest = e2e.DefaultShoot("e2e-upgrade-hib", false)
```

Comment:
Why didn't you introduce workerless tests in this file as well?

Reply:
We cannot create a shoot in the previous release, since the `WorkerlessShoots` featuregate is turned on only in this PR.

Reply:
No, I will add a task in the issue.